packages/@ourworldindata/grapher/src/mapCharts/ChoroplethGlobe.tsx

```diff
@@ -0,0 +1,622 @@
+            </g>
+        )
+    }
+
+    renderFeaturesWithData(): React.ReactElement | void {
+        if (this.featuresWithData.length === 0) return
+
+        return (
+            <g id={makeIdForHumanConsumption("countries-with-data")}>
+                {this.featuresWithData.map((feature) => {
+                    const series = this.choroplethData.get(feature.id)
+                    if (!series) return null
+                    return (
+                        <CountryWithData
+                            key={feature.id}
+                            feature={feature}
+                            series={series}
+                            path={this.getPath(feature)}
+                            focus={this.getFocusState(feature.id)}
+                            showSelectedStyle={this.showSelectedStyle}
+                            onClick={(event) =>
+                                this.onClick(feature, event.nativeEvent)
+                            }
+                            onTouchStart={() => this.onTouchStart(feature)}
+                            onMouseEnter={this.onMouseEnterFeature}
+                            onMouseLeave={this.onMouseLeaveFeature}
+                        />
+                    )
+                })}
+            </g>
+        )
+    }
+
+    renderStatic(): React.ReactElement {
+        return (
+            <>
+                {this.renderGlobeOutline()}
+                <g id={makeIdForHumanConsumption("globe")}>
+                    {this.renderFeaturesWithNoData()}
+                    {this.renderFeaturesWithData()}
+                </g>
+            </>
+        )
+    }
+
+    renderInteractive(): React.ReactElement {
+        // this needs to be referenced here or it will be recomputed on every mousemove
+        const _cachedCentroids = this.quadtree
+
+        return (
+            <g
+                ref={this.base}
+                onMouseDown={
+                    (ev: SVGMouseEvent): void =>
+                        ev.preventDefault() /* Without this, title may get selected while shift clicking */
+                }
+                onMouseMove={(ev: SVGMouseEvent): void =>
+                    this.onMouseMove(ev.nativeEvent)
+                }
+                onMouseLeave={this.onMouseLeaveFeature}
+                style={{ cursor: this.hoverFeature ? "pointer" : undefined }}
+            >
+                {this.renderGlobeOutline()}
+                <g className={GEO_FEATURES_CLASSNAME}>
+                    {this.renderFeaturesWithNoData()}
+                    {this.renderFeaturesWithData()}
+                </g>
+            </g>
+        )
+    }
+
+    render(): React.ReactElement {
+        return this.manager.isStatic
+            ? this.renderStatic()
+            : this.renderInteractive()
+    }
+}
+
+const isMultiTouchEvent = (
+    event: MouseEvent | TouchEvent
+): event is TouchEvent => {
+    return checkIsTouchEvent(event) && event.touches.length >= 2
```

Comment:
We should probably only make it a pinch if it's _exactly two_ touches?

Reply:
I don't think it matters much, but I can change it